src/tracer/tracee_memory.rs

```diff
@@ -104,13 +121,26 @@ mod peeking {
             [0, 0, 0, 0, 0, 0, 0, 0],
         ]
         .into_iter()
-        .map(cast_to_word)
+        .map(cast_to_eight_byte_word)
         .map(Ok);
         assert_eq!(data_to_string(data).unwrap(), b"abcdefgh");
     }
 }
 
-fn pokedata(pid: Pid, address: c_ulonglong, words: c_ulonglong) -> R<()> {
+pub fn poke_four_bytes(pid: Pid, address: c_ulonglong, small_word: i32) -> R<()> {
```

Comment:
Maybe for consistency with how the words are passed in change this from `i32` to `c_uint`? (That's an alias for `u32`.)